rpm/entry.sh

```diff
@@ -1,11 +1,39 @@
-#!/bin/sh
+#!/bin/bash
 # Entrypoint for the build container to create the rpms and yum repodata:
 
 set -e
-# Download sources if not already available
-cd /root/rpmbuild/SPECS  && spectool -gf kubelet.spec
 
-/usr/bin/rpmbuild --define "_sourcedir /root/rpmbuild/SPECS/" -bb /root/rpmbuild/SPECS/kubelet.spec
+declare -a ARCHS
 
-mkdir -p /root/rpmbuild/RPMS/x86_64/
-createrepo -o /root/rpmbuild/RPMS/x86_64/ /root/rpmbuild/RPMS/x86_64
+if [ $# -gt 0 ]; then
```

Comment:
Do it either way; bash scripts are kind of terrible for this task in any case.
I expect this to be merged with debian/build.go pretty soon anyway...